src/main/java/org/zalando/nakadi/domain/NakadiCursorLag.java

```diff
@@ -0,0 +1,57 @@
+package org.zalando.nakadi.domain;
+
+public class NakadiCursorLag {
+    private String topic;
+    private String partition;
```

Comment:
What is the point of information duplication? I mean even if topic is needed here, then there are 2 cursors there, that should contain the same information.